### ISSUE TITLE
aria-label of status contains body even if it has a spoiler

### ISSUE THREAD

**Issue by akihikodaki** [tootsuite/mastodon#5192]
`aria-label` is in format of `spoiler content`, where content should be hidden.
https://github.com/tootsuite/mastodon/commit/c71874b84c1616f2a55547e1aa7cf81fc5c30636#diff-a2a27340bfd3f4bbb805f05a8776c4c2L149
@sorin-davidoi I'd like to hear the intension of the label.

* * * *

- [X] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
- [ ] This bug happens on a [tagged release](https://github.com/tootsuite/mastodon/releases) and not on `master` (If you're a user, don't worry about this).

**Comment by sorin-davidoi** [comment]
In hindsight, I think it may not be needed, as the screen readers can access the inside text anyway.
